### ISSUE TITLE
Simple typo in doc/core/development/policy/coding-standard.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6674]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#6674|
|Type|enhancement|
|Created|2013-08-08 09:20:11Z|
|Branch|https://github.com/twisted/twisted/tree/fix-typo-in-coding-standard-6674|

`It is preferred that every block appear on a new line, […]`

should be:

`It is preferred that every block appears on a new line,  […]`

<details><summary>Searchable metadata</summary>

```
trac-id__6674 6674
type__enhancement enhancement
reporter__hynek hynek
priority__normal normal
milestone__ 
branch__branches_fix_typo_in_coding_standard_6674 branches/fix-typo-in-coding-standard-6674
branch_author__hynek hynek
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1375953611000000 1375953611000000
changetime__1375956931000000 1375956931000000
version__None None
owner__hynek hynek

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_1"></a>|@hynek commented|
|-|-|

(In [39534]) Branching to fix-typo-in-coding-standard-6674.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)|@hynek removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @hynek|
|-|-|

Cool, please merge!

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_4"></a>|@hynek set status to `closed`|
|-|-|

(In [39536]) Merge fix-typo-in-coding-standard-6674: Fix typo in coding standard docs

Author: hynek
Reviewers: therve
Fixes: [#6674](https://github.com/twisted/twisted/issues/6674)

Fix a grammar typo.
